data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -282,6 +306,7 @@ public void testEqualityDeletesSpanningMultipleDataFiles() throws IOException {
         FileHelpers.writeDataFile(table, Files.localOutput(temp.newFile()), Row.of(0), records);
 
     table.newAppend().appendFile(dataFile).commit();
+    // At this point, the table has 7 + 8 = 15 rows, of which all but one are in duplicate.
```

Comment:
Nit / non-blocking: it’s a little strange to me to have the whole line comments with empty lines afterwards.

Since this is just a test and it said “at this point”, I feel like we could move it to be directly above the next code block. But it’s just test code so I absolutely wouldn’t block on this.

Reply:
Perhaps I'm idiosyncratic. I put the comment after the statement because the "at this point" refers to the state after the statement has executed and not before. (If the comment were short enough, I would have put it on the same line as the statement, after it. That would probably be more natural.)

Nevertheless, I have now put the comments before the statements.